### ISSUE TITLE
Problems with PTC-Account already exists

### ISSUE THREAD

**Issue by MiguelTellezRuiz** [drallieiv/KinanCity#71]
There is a problem with your email address. Please verify that it is spelled correctly. If it is not, please correct it and try again. If you already have a Pok&eacute;mon Trainer Club account with this email address, please use the Forgot Password Tool to find your account.

I have this message always.

**Comment by drallieiv** [comment]
Well are you sure that there is not already a PTC account with that same email like the message says? It is usually the case when you get that error.

**Comment by MiguelTellezRuiz** [comment]
Yes, im pretty sure about this. I tried a lot of times with different acounts emails, and I have this response all times. Actually, is it working correct? If the answer is yes I'll try with differents parameters.

**Comment by drallieiv** [comment]
Please DM one of you log file on discord so I can check it out.

**Comment by chancity** [comment]
That happens when a domains mx/a records aren’t properly configured.
